### ISSUE TITLE
Traceback 

### ISSUE THREAD

**Issue by SaikotykSoul** [DIGITALCRIMINALS/OnlyFans#462]
Hi. I have no idea what I'm doing, but I followed a guide and I came up with the following when trying to run the start_ofd.py file after I set up the config.json and the auth.json. Any help or guidance would be greatly appreciated. Please ELI5.

Traceback (most recent call last):
  File "C:\Users\ashto\Downloads\OnlyFans-7.6.1\start_ofd.py", line 104, in <module>
    asyncio.run(main())
  File "C:\Users\ashto\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\ashto\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 646, in run_until_complete
    return future.result()
  File "C:\Users\ashto\Downloads\OnlyFans-7.6.1\start_ofd.py", line 86, in main
    api = await main_datascraper.start_datascraper(
  File "C:\Users\ashto\Downloads\OnlyFans-7.6.1\datascraper\main_datascraper.py", line 58, in start_datascraper
    auths = choose_option(subscription_list, auto_profile_choice, True)
  File "C:\Users\ashto\Downloads\OnlyFans-7.6.1\helpers\main_helper.py", line 901, in choose_option
    if value.isdigit():
AttributeError: 'int' object has no attribute 'isdigit'

**Comment by SaikotykSoul** [comment]
Nevermind! It was an authentication issue! Thanks! XD

**Comment by Bradwii** [comment]
> 


> Nevermind! It was an authentication issue! Thanks! XD

Hey, I'm having the same problem, how did you fix this?
